src/settings-ui/Microsoft.PowerToys.Settings.UI.Library/EnabledModules.cs

```diff
@@ -80,7 +80,7 @@ public bool ShortcutGuide
             }
         }
 
-        private bool videoConference = true;
+        private bool videoConference; // defaulting to off https://github.com/microsoft/PowerToys/issues/14507
```

Comment:
I suggest to explicit use `=false`. It's more save against changes in null value Interpretation of the module interface.

Reply:
A c# bool can’t be null unless it is defined as a nullable type. This is not so this is correct syntax (also style cop gets mad)